components/nested_star/Cargo.toml

```diff
@@ -0,0 +1,19 @@
+[package]
+name = "nested-star-cxx"
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
+nested-sta-rs = { path = "../../../../../nested-sta-rs" }
```

Comment:
This should point to a specific git revision instead of a parallel checkout.

Reply:
commit c13dd394 already fixes this

Reply:
Ok, thanks.